CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Solidus 2.7.0 (2018-09-14)
 
+### Core
```

Comment:
Wait, this should stay under 2.8.0 instead!

Reply:
Whoops, fixing!

Reply:
@kennyadsl this was fixed, can you check?